SIG.md

```diff
@@ -0,0 +1,85 @@
+Leads MAY select additional Leads through a super-majority vote amongst Leads. This SHOULD be supported by a majority of SIG contributors.
+
+## Other Roles
+This governance document outlines the required roles for SIGs; however, SIGs are allowed to operate how they see fit outside of minimum governance requirements, including defining more roles to sustain the group. If a SIG needs to change the Lead position to include or remove duties, this needs to be approved by the committers. Newly created roles that don't assume any responsibility of Leads should follow the governing processes in the SIG's charter.
+
+# SIG Creation
+
+## Goals
+The primary goal of the charters is to define the scope of the SIG within Cilium and how the SIG leaders exercise ownership of these areas by taking care of their responsibilities. A majority of the effort should be spent on these concerns. SIG Leads are also reponsible for driving reviews for CFPs for their focus area, and have a responsibility to bring concerns to committers if they are unsure. 
+
+### SIG Charter Guide
+All Cilium SIGs must define a charter defining the scope and governance of the SIG.
+The scope must define what areas the SIG is responsible for directing and maintaining.
+The governance must outline the responsibilities within the SIG as well as the roles owning those responsibilities.
+
+### Steps to create a SIG charter
+1. Copy the [template](./sig-TEMPLATE/charter.md) into a new file under community/sig-YOURSIG/charter.md
+1. Read the recommendations and requirements so you have context for the template
+1. Fill out the template for your SIG
+1. Update sigs.yaml with the individuals holding the roles as defined in the template.
+1. Create a pull request with a draft of your charter.md and sigs.yaml changes. Communicate it within your SIG and get feedback as needed.
+1. Send the SIG Charter out for review to the Cilium committers.
+1. Once accepted, the committers will ratify the PR by merging it.
+
+### Steps to update an existing SIG charter
+For significant changes, or any changes that could impact other SIGs, such as the scope, create a PR and send it to the committers for review with the subject: "SIG Charter Update: YOURSIG".
+For minor updates to that only impact issues or areas within the scope of the SIG the SIG Leads should facilitate the change.
+
+### SIG Charter approval process
+When introducing a SIG charter or modification of a charter the following process should be used.
+
+1. Identify a small set of people from the SIG to drive the changes. Most typically this will be the SIG Leads.
+1. Work with the rest of the SIG in question to craft the changes. Make sure to keep the SIG in the loop as discussions progress with the committers.
+1. Work with the committers to gain approval. This can simply be submitting a PR and sending a slack message. If more substantial changes are desired it is advisable to socialize those before drafting a PR.
```

Comment:
I think this is probably implied, but "socialize" is a bit vague. An example might help.


```suggestion
1. Work with the committers to gain approval. This can simply be submitting a PR and sending a slack message. If more substantial changes are desired it is advisable to socialize those before drafting a PR, for instance by connecting with a committer to act as a mentor and sponsor for this process.
```